cmd/gomigen/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/Makpoc/gomigen/internal/app"
+)
+
+func main() {
+	outputDir := flag.String(
+		"out",
+		"", // default is current dir
+		"optional path to output directory where the generated package with middleware files will be placed",
+	)
+	flag.Parse()
+
+	// TODO would it be better to make all args named?
```

Comment:
I think it is fine as it is.

Reply:
Also, you can consider placing such questions into the Issue section maybe so that they can easily be discussed upon later on?

Reply:
I removed that particular TODO, but I may move the TODOs from the README there some day.